setup.cfg

```diff
@@ -89,6 +89,7 @@ test =
     pytest-cov
     pytest-mock
     responses
+    tox
```

Comment:
This does not belong here; for one thing, if you're installing tox, then you don't want to install the other test dependencies in the same environment, as tox will install them in one of its environments.

Reply:
ok, that's true. But do you still suggest that people use `tox` to test the source code locally (instead of `pytest` directly)? Should I include `pip install tox` in `DEVELOPMENT.md`?

Reply:
Yes.